contracts/core/Controller.sol

```diff
@@ -169,21 +169,23 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
     );
     /// @notice emits an event when a vault is settled
     event VaultSettled(
-        address indexed AccountOwner,
-        address indexed to,
-        address indexed otoken,
+        address indexed accountOwner,
+        address indexed oTokenAddress,
+        address payoutRecipient,
+        uint256 payout,
         uint256 vaultId,
-        uint256 payout
+        uint256 indexed vaultType
     );
     /// @notice emits an event when a vault is liquidated
     event VaultLiquidated(
         address indexed liquidator,
-        address indexed receiver,
+        address indexed payoutRecipient,
```

Comment:
Same here, can we keep it receiver to match the same name used in the action argument.

Reply:
Are you open to renaming it to `payoutRecipient`? It's more explicit.